sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala

```diff
@@ -110,6 +110,13 @@ package object dsl {
     def isNull: Predicate = IsNull(expr)
     def isNotNull: Predicate = IsNotNull(expr)
 
+    def isTrue: Predicate = IsTrue(expr)
+    def isNotTrue: Predicate = IsNotTrue(expr)
+    def isFalse: Predicate = IsFalse(expr)
+    def isNotFalse: Predicate = IsNotFalse(expr)
+    def isUnknown: Predicate = IsUnknown(expr)
+    def isNotUnknown: Predicate = IsNotUnknown(expr)
```

Comment:
Sorry for adding a comment for the late review rounds, but shall we avoid adding DSL since the whole picture is to support PostgreSQL feature parity? How do you think about that, @maropu ? Can we remove these DSL addition?

Reply:
In that case, we should revert this and `test("is true | false | unknown expressions")`.

Reply:
I don't have a strong opnion on this though, +1 for your opnion. I don't think all the syntaxes need to be listed up here (It is enough to list up a basic part of them).

Reply:
@dongjoon-hyun This is not only a PostgreSQL feature, but a ANSI SQL. I also want to confirm if I really want to delete this and `test("is true | false | unknown expressions")`.

Reply:
No, @beliefer . You are confused with Scala DSL and SQL.
This is `sql/catalyst/dsl` which providing Scala API. This is never ANSI SQL.

Reply:
> No, @beliefer . You are confused with Scala DSL and SQL.
> This is `sql/catalyst/dsl` which providing Scala API. This is never ANSI SQL.
> SQL Standard doesn't have `def isNotUnknown`. :)

Thanks for your prompt. I am confused indeed and clear now.